reconcile/slack_usergroups.py

```diff
@@ -56,23 +57,33 @@
           scheduleID
           escalationPolicyID
         }
-        github_owners
-        gitlab_owners
         channels
         description
-        github_owners_aliases
-        gitlab_owners_aliases
+        ownersFromRepo
       }
     }
   }
 }
 """
 
 
+class RepoCli:
```

Comment:
It looks a bit strange that `RepoCli` objects are `GitxxxApi` objects under the hood... Maybe `ProviderApi` or `RepoProviderApi`? Naming is painful, it is ok if you prefer your name :-D

Reply:
you're right :)